packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -739,10 +739,8 @@ contract BridgePool is MultiCaller, Testable, BridgePoolInterface, ERC20, Lockab
         _sync(); // Fetch any balance changes due to token bridging finalization and factor them in.
 
         // ExchangeRate := (liquidReserves + utilizedReserves - undistributedLpFees) / lpTokenSupply
-        uint256 numerator = liquidReserves - undistributedLpFees;
-        if (utilizedReserves > 0) numerator += uint256(utilizedReserves);
-        else numerator -= uint256(utilizedReserves * -1);
-        return (numerator * 1e18) / totalSupply();
+        int256 numerator = int256(liquidReserves) + utilizedReserves - int256(undistributedLpFees);
```

Comment:
What gives me confidence is that this is just a refactor of the above lines where you cast immediately to `int256` to avoid any underflow possibilities. Nice!